### ISSUE TITLE
Hides blocked sites in console

### ISSUE THREAD

**Issue by rugk** [mozilla/blok#188]
Without this extension, when the tracking mode is activated (e.g. via `about:config`) I could see the blocked sites in the developer console.
However now they are hidden!

This is bad as I now cannot determinate what services sites where trying to use.

**Comment by groovecoder** [comment]
Thanks for filing the issue. Labeling ...

**Comment by jryans** [comment]
I agree this is important feature to have. It's quite hard to understand what has been blocked by tracking protection when you don't have the console warnings.
